climpred/prediction.py

```diff
@@ -276,12 +270,12 @@ def compute_hindcast(
                 f'dim={dim} | '
                 # This is the init-sliced forecast, thus displaying actual
                 # initializations.
-                f'inits={forecast["time"].min().values}'
-                f'-{forecast["time"].max().values} | '
+                f'inits={inits[i].min().values}'
+                f'-{inits[i].max().values} | '
```

Comment:
didnt you want to add all times instead of min max only? or at least at the debug level?

Reply:
That will be done on the debug level in the next PR. I just want to get this one in so I can work on `maximize`. I'll put it on the list for maximize, though.